files/chef-marketplace-ctl-commands/show_instance_id.rb

```diff
@@ -0,0 +1,20 @@
+require "ohai"
+
+add_command_under_category "show-instance-id", "Configuration", "Show the cloud instance ID. Required for first time setup authorization.", 2 do
+  puts "Looking up current Instance ID..."
+
+  system = Ohai::System.new.all_plugins
```

Comment:
There is probably a way to run just the Azure plugin and shave of a couple seconds, but I couldn't figure out an easy way to do this. :man_shrugging: